Proposta de novo Modelo para o CONTRIBUTING (#230)
OpenDevUFCG/Tamburetei: CONTRIBUTING.md

> Contribuições sempre serão bem vindas, sejam pequenas ou grandes. Leia o [código de conduta](https://github.com/OpenDevUFCG/Tamburetei/blob/master/CODE_OF_CONDUCT.md) antes de abrir qualquer issue ou pull request.

Reviewer: Já que foi explicado na seção seguinte, creio que não é mais necessário citar o `CODE_OF_CONDUCT.md` aqui.

Author: Queria uma frase bonitinha pra botar no lugar kkkkk

## Código de Conduta

> Adotamos um [código de conduta](https://github.com/OpenDevUFCG/Tamburetei/blob/master/CODE_OF_CONDUCT.md) que esperamos que todos sigam a participar do projeto. Leia para entender que ações toleramos e quais não antes de fazer qualquer contribuição.

Reviewer 2: Acho que assim ficaria melhor:

> Adotamos um [código de conduta](https://github.com/OpenDevUFCG/Tamburetei/blob/master/CODE_OF_CONDUCT.md) e esperamos que todos os participantes de nossos projetos o sigam. Antes de fazer qualquer contribuição, é importante que você conheça quais ações não são toleradas em nossa comunidade.

## Organização dos diretórios

> Todas as pastas de disciplinas seguem uma estrutura padrão para facilitar entendimento e manutenção. Leia essa breve [explicação](https://github.com/OpenDevUFCG/Tamburetei/wiki/Organiza%C3%A7%C3%A3o-de-Diret%C3%B3rios) para entender como funciona.

Reviewer 2: Também mudaria essa parte, acho que fica mais compreensível.

> Visando facilitar o entendimento e a manutenção do Tamburetei, utilizamos uma estrutura padrão nos diretórios das disciplinas. Para compreendê-la melhor, leia essa breve [explicação](https://github.com/OpenDevUFCG/Tamburetei/wiki/Organiza%C3%A7%C3%A3o-de-Diret%C3%B3rios).

## Organização de Issues

> As issues são um espaço aberto para requisitar criação, mudança ou correção de material. Também pode ser usada como um espaço de discussão sobre as cadeiras ~tamburetes~ de Ciência da Computação da UFCG. Tenha em mente que o tema sempre deve estar relacionado ao curso.

Reviewer 2: Trocaria "cadeiras" por "disciplinas". Apesar de ser uma gíria bem comum, ainda existem pessoas que não usam esse termo e isso pode atrapalhar a compreensão, inclusive para pessoas de outras universidades que queiram conhecer a gente.

> Todas as pastas de disciplinas seguem uma estrutura padrão para facilitar entendimento e manuntenção. Leia essa breve [explicação](https://github.com/OpenDevUFCG/Tamburetei/wiki/Organiza%C3%A7%C3%A3o-de-Diret%C3%B3rios) para entender como funciona.

Reviewer: ```suggestion
Todas as pastas de disciplinas seguem uma estrutura padrão para facilitar entendimento e manutenção. Leia essa breve [explicação](https://github.com/OpenDevUFCG/Tamburetei/wiki/Organiza%C3%A7%C3%A3o-de-Diret%C3%B3rios) para entender como funciona.
```

Veja sobre nossa [organização de Issues](https://github.com/OpenDevUFCG/Tamburetei/wiki/Organiza%C3%A7%C3%A3o-de-Issues)

## Propondo uma mudança

> Caso deseje propor uma mudança, primeiro verifique se já não existe uma Issue sobre o que você deseja. Se não, [abra uma Issue](https://github.com/OpenDevUFCG/Tamburetei/issues/new) seguindo o *template* adequeado ao seu problema e espere a comunidade te responder.

Reviewer 2: adequado *

> As issues são um espaço aberto para requisitar criação, mudança ou conserto de material. Também pode ser usada como um espaço de discussão sobre as cadeiras ~tamburetes~ de Ciência da Computação da UFCG. Tenha em mente que o tema sempre deve estar relacionado ao curso.

Reviewer: ```suggestion
As issues são um espaço aberto para requisitar criação, mudança ou correção de material. Também pode ser usada como um espaço de discussão sobre as cadeiras ~tamburetes~ de Ciência da Computação da UFCG. Tenha em mente que o tema sempre deve estar relacionado ao curso.
```

## Primeira contribuição

> Caso tenha se interessado pela nossa comunidade, deseja contribuir, mas não sabe por onde começar, dê uma olhada em issues com a tag [`good first issue`](https://github.com/OpenDevUFCG/Tamburetei/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22) que serão um bom ponto de partida. Também vale a pena olhar sobre Issues relacionadas a cadeiras que você já pagou ou está pagando.

Reviewer 2: Outra proposta de alteração, inclusive relacionada ao comentário anterior.

> Se você se interessou pela nossa comunidade e deseja contribuir conosco, mas não sabe por onde começar, dê uma olhada em Issues com a tag [`good first issue`](https://github.com/OpenDevUFCG/Tamburetei/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22), elas serão um bom ponto de partida. Também vale a pena olhar as Issues relacionadas às disciplinas que você já cursou ou está cursando.

> Caso esteja procurando no que contribuir, dê uma olhada em issues com a tag [`good first issue`](https://github.com/OpenDevUFCG/Tamburetei/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22) que serão um bom ponto de partida. Também vale a pena olhar sobre Issues relacionadas a cadeiras que você já pagou ou está pagando.

Reviewer: ```suggestion
Caso tenha se interessado pela nossa comunidade, deseja contribuir, mas não sabe por onde começar, dê uma olhada em issues com a tag [`good first issue`](https://github.com/OpenDevUFCG/Tamburetei/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22) que serão um bom ponto de partida. Também vale a pena olhar sobre Issues relacionadas a cadeiras que você já pagou ou está pagando.
```

> Se você decide fazer alguma mudança, e não existe uma Issue sobre, crie uma nova seguindo o *template* adequeado informando o que será feito e manifeste que você irá querer fazer. Caso exista a Issue, verifique nos comentários se já não existe alguém trabalhando nisso, e comente que você deseja fazer.

Reviewer 2: Os templates já são citados anteriormente, remover a menção torna o texto mais compreensível. Também mudei um pouco para explicitar que mais de uma pessoa pode trabalhar em uma issue.

> Se você decide fazer alguma mudança e ainda não existe uma Issue sobre esse tema, crie uma nova para informar o que será feito e para manifestar seu interesse em atender essa demanda. Caso a Issue já exista, manifeste nos comentários seu interesse em contribuir para a resolução dela. Mesmo que alguém já esteja trabalhando para isso, vocês podem sincronizar seus esforços!

> Não tenha medo de pedir por orientação ou ajuda, mas, acima de tudo, seja corajoso e respeitoso. Você também pode participar de discussões para ajudar a comunidade a realizar boas escolhas!

Reviewer 2: Acho que o *"corajoso"* foi uma tradução pra *"bold"*, creio que destemido se adequa melhor ao sentido.

> Não tenha medo de pedir por orientação ou ajuda, o importante é ser destemido e respeitoso! Você também pode participar de discussões para ajudar nossa comunidade a realizar boas escolhas!

## Restrições de Upload

> Contemos algumas [restrições](https://github.com/OpenDevUFCG/Tamburetei/wiki/Restri%C3%A7%C3%B5es-de-Upload) de upload de arquivos ao repositório que nos ajudam a manter o projeto sustentável, por favor leia antes de submeter qualquer arquivo.

Reviewer: ```suggestion
Devido a problemas de copyright e limite de armazenamento para repositórios do Github, algumas [restrições](https://github.com/OpenDevUFCG/Tamburetei/wiki/Restri%C3%A7%C3%B5es-de-Upload) de upload de arquivos ao repositório que nos ajudam a manter o projeto sustentável, por favor leia antes de submeter qualquer arquivo.
```

> Devido a problemas de copyright e limite de armazenamento para repositórios do Github, algumas [restrições](https://github.com/OpenDevUFCG/Tamburetei/wiki/Restri%C3%A7%C3%B5es-de-Upload) de upload de arquivos ao repositório que nos ajudam a manter o projeto sustentável foram adotadas. Por favor leia antes de submeter qualquer arquivo.

Reviewer 2: Uma reformuladinha para deixar a primeira frase mais direta/menos extensa.

> Para manter o Tamburetei sustentável, possuímos algumas [restrições](https://github.com/OpenDevUFCG/Tamburetei/wiki/Restri%C3%A7%C3%B5es-de-Upload) sobre o *upload* de arquivos em nosso repositório, sobretudo em relação a *copyright* e limites de armazenamento. Esteja ciente dessas restrições antes de submeter qualquer arquivo!
